### DISCUSSION TITLE
Autocomplete names when :w myN{tab}

### DISCUSSION THREAD

**Discussion by freddieventura** [andmarti1424/sc-im#666] [Ideas]
I'm just missing the feature of pressing Tab , say I got a file `myFileLongName.csv` and I'm on the same path and I want to create an alternative out of the firstone `myFileLongName02.csv` I would just `:w myFile{tab}` but that is not autocompleting.

Thanks

**Comment by andmarti1424** [comment]
Please open discussions for feature requests. Already move this to there.

**Comment by andmarti1424** [comment]
What u suggest would be indeed useful.
